runtime/indent/mojo.vim

```diff
@@ -0,0 +1,15 @@
+" Vim indent file
+" Language:	Mojo
+" Maintainer:	Riley Bruins <ribru17@gmail.com>
+" Last Change:	2024 Jul 07
+
+" Only load this indent file when no other was loaded.
+if exists("b:did_indent")
+   finish
+endif
+let b:did_indent = 1
+
+setlocal cinkeys-=0#
+setlocal indentkeys-=0#
```

Comment:
I suppose you would also want `:setl cindent`?

Reply:
I'm not sure; these were taken verbatim from the python `ftplugin`, but python also has it's own conflicting indent settings in its indent plugin (which also sets `indentexpr` rather than `cindent`). Perhaps mojo should just inherit this indent file and call it a day? (Mojo syntax is a superset of python's)

Reply:
Sure, you can alternatively just source the python indent script if that works.